framework/utils/frontUtils.py

```diff
@@ -22,7 +22,7 @@
 
 
 
-def nonDominatedFrontier(data, returnMask):
+def nonDominatedFrontier(data, returnMask, minMask=None):
```

Comment:
@wangcj05 I honestly feel like the required structure of returnMask and minMask might not be optimal, if you have comments/suggestions let me know

Reply:
Sorry, I did not get your point here. Could you clarify it?

Reply:
I was not sure if passing the data as numpy array and min mask as a separate numpy array was the best way to pass this info to this method